cinema_service/settings.py

```diff
@@ -44,11 +34,11 @@
     "django.contrib.staticfiles",
     "rest_framework",
     "rest_framework.authtoken",
+    "drf_spectacular",
     "debug_toolbar",
     "cinema",
     "user",
 ]
-
 MIDDLEWARE = [
     "django.middleware.security.SecurityMiddleware",
     "debug_toolbar.middleware.DebugToolbarMiddleware",
```

Comment:
Ensure that the debug toolbar is only included in development environments. You can wrap this middleware with a condition to check if `DEBUG` is `True`.